Tiltfile

```diff
@@ -0,0 +1,93 @@
+load("ext://restart_process", "docker_build_with_restart")
+load("ext://helm_resource", "helm_resource", "helm_repo")
+load("ext://configmap", "configmap_create")
+
+# A list of directories where changes trigger a hot-reload of PATH.
+# Note: this list needs to be updated each time a new package is added to the repo.
+hot_reload_dirs = ["cmd", "config", "gateway", "health", "message", "qos", "relayer", "request", "router"]
+
+# Load the existing config file, if it exists, or use an empty dict as fallback
+localnet_config_path = "localnet_config.yaml"
+localnet_config = read_yaml(localnet_config_path, default={})
+
+# TODO_UPNEXT(@adshmh): Package the default Helm chart for PATH and upload the pokt-network repo.
```

Comment:
`Package the default Helm chart for PATH`

Very unclear to me where this will come from and what it is. Specifically

1. @red-0ne is doing things to make PATH work in poktroll
2. @HebertCL is building helm-chars to deploy PATH to staging
3. @okdas might have context but not the DRI

IMO, (2) is what we should be * reusing * here.